foreman.toml

```diff
@@ -0,0 +1,6 @@
+[tools]
+luau-lsp = { source = "JohnnyMorganz/luau-lsp", version = "1.18.1" }
```

Comment:
I'd rather be more explicit and intentional here, otherwise someone making a PR could "lose" the type checking lottery and be semi-forced to deal with new false (or true) positives in Luau releases.